docs/CONTRIBUTING.md

```diff
@@ -268,6 +268,15 @@ postgres`). Generally, the problem is that by default the postgres server is
 > we'll help fix the problem and will add the solution to these
 > instructions!
 
+Another options is to use a standalone Docker container for Postgres.
+
+```sh
+# example using postgres 15.15-trixie
+docker run -e POSTGRES_PASSWORD=password -p 5432:5432 postgres:15.15-trixie
```

Comment:
we are using Postgres 16 in production, so these instructions should probably reflect that